01/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+¿Qué es GitHub?
+Es un repositorio online gratuito que permite gestionar proyectos y controlar versiones de código. Es muy utilizado por desarrolladores para almacenar sus trabajos dando así la oportunidad a millones de personas de todo el mundo a cooperar en ellos.
+Permite alojar proyectos utilizando el sistema de control de versiones Git. 
+¿Para qué sirve?
+Se utiliza principalmente para la creación de código fuente de programas de ordenador.
+¿Qué es un PR?
+Una Pull request funciona para solicitar que otro desarrollador (por ejemplo, el mantenedor del proyecto) incorpore (o haga un pull) una rama de tu repositorio al suyo.
+¿Qué es un merge?
+Permite tomar las líneas independientes de desarrollo creadas por git branch e integrarlas en una sola rama. 
```

Comment:
Again, try using your own words. We don't want a dictionary definition. We want to know if you understood the main idea.